packages/loader/container-loader/src/container.ts

```diff
@@ -555,6 +556,13 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     private get scope() { return this.loader.services.scope; }
     private get codeLoader() { return this.loader.services.codeLoader; }
 
+    /**
+     * {@inheritDoc @fluidframework/container-definitions#IContainer.entryPoint}
+     */
+    public get entryPoint() : Promise<FluidObject | undefined> {
+        return this.context.entryPoint;
```

Comment:
i think `this.context` will throw if the context isn't yet defined. i think you can make this better:
```
while(this._context === undefined){
    await new Promise((resolve,reject)=>{
        this.once("contextChanged", resolve);
        this.once("closed", reject);
    });
}
return this._context;
```